Source/WebKit/WebProcess/GPU/graphics/RemoteResourceCacheProxy.cpp

```diff
@@ -137,10 +137,14 @@ void RemoteResourceCacheProxy::recordNativeImageUse(NativeImage& image)
 
 void RemoteResourceCacheProxy::recordFontUse(Font& font)
 {
+    if (font.platformData().customPlatformData())
+        recordFontCustomPlatformDataUse(*font.platformData().customPlatformData());
```

Comment:
I guess this is fine. The crux of this set of patches is that you're splitting a font into 2 objects and serializing them separately, with different lifetimes, which naively seems unintuitive and overengineered, but if the memory savings make it worth it, then I guess it's fine. The repurposing of `FontCustomPlatformData` into a retained object that gets IPC'ed around, rather than just an implementation detail of how fonts get created internally, is a bit scary, but I can't think of a better way to do it without a major architecture change.